regen-network/regen-web
# Fix infinite render cycle - credit class page

## Description

Closes: #1380 

- Credit Class page was crashing for C02 in prod. This removes a gate that was resulting in an infinite render loop.
- Handles some cases where data was in a previous shacl shape for C02 class.



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


#### Setting up backport PRs

After merging your PR to `master`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR, `@Mergifyio backport redwood hambach`.

2. If your branch does have merge commits:

    a. Pull latest `master`, `hambach` and `redwood` branches

    b. Create new branches for backports and merge `master` (replace `<PR#>` with your PR #)
    ```
    git checkout -b hambach-backport-<PR#> hambach
    git merge master
    git push origin hambach-backport-<PR#>
    git checkout -b redwood-backport-<PR#> redwood
    git merge master
    git push origin redwood-backport-<PR#>`
    ```

    c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.
  

### How to test

1. https://deploy-preview-1382--regen-registry.netlify.app/credit-classes/C02 should render PROD/Mainnet C02 data (and not crash). _Note* Need to revert change to netlify.toml before merging_

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
